src/main/java/io/javalin/metrics/JavalinMetrics.java

```diff
@@ -0,0 +1,32 @@
+package io.javalin.metrics;
```

Comment:
I saw the licence header in on of your other files. Should I just copy it from another source file?

Reply:
Yeah, that should be fine.

Reply:
done